ckanext/ytp/comments/templates/package/read_base.html

```diff
@@ -1,8 +1,10 @@
 {% ckan_extends %}
 
 {% block content_primary_nav %}
-    {{ super() }}
```

Comment:
I'm not a fan of fully replacing the base page by removing super.
which at the beginning is
https://github.com/ckan/ckan/blob/64774670b722ed1bd3808901137fa80753d3c772/ckan/templates/package/read_base.html#L20

so my question is:
what happens if the plugins are re-ordered and content_primary_nav is changed by another plugin before this?

looking at https://flask.palletsprojects.com/en/1.1.x/patterns/templateinheritance/?highlight=super
there seems to not be any way to edit ordering of something.

can we change it from position 2 to position 4 (usually after activity).

Reply:
This is why we put the override for `content_primary_nav` in `ckanext-data-qld-theme` and left the override in `ckanext-ytp-comments` as generic, using `{{ super() }}` - feels like we're going in circles now?